skbase/base/_tagmanager.py

```diff
@@ -0,0 +1,215 @@
+# -*- coding: utf-8 -*-
+"""Mixin class for flag and configuration settings management."""
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+__author__ = ["fkiraly"]
+__all__ = ["_FlagManager"]
+
+
+import inspect
+from copy import deepcopy
+
+
+class _FlagManager:
+    """Mixin class for flag and configuration settings management."""
+
+    @classmethod
+    def _get_class_flags(cls, flag_attr_name="_flags"):
+        """Get class flags from estimator class and all its parent classes.
+
+        Parameters
+        ----------
+        flag_attr_name : str, optional, default = "_flags"
```

Comment:
Minor tweak, but [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#sections) notes that if the default value is used then you don't put "optional" part of parameter docstring (e.g., you'd just put flag_attr_name : str, default="_flags").

They also show no spaces around equal sign.

Can you take a quick look at the docstrings in the PR and update these.

Reply:
I see, you would use "optional" only if you have sth like `None`. Fixed in all parallel cases